src/workerd/api/global-scope.h

```diff
@@ -386,6 +404,7 @@ class ServiceWorkerGlobalScope: public WorkerGlobalScope {
     JSG_LAZY_INSTANCE_PROPERTY(crypto, getCrypto);
     JSG_LAZY_INSTANCE_PROPERTY(caches, getCaches);
     JSG_LAZY_INSTANCE_PROPERTY(scheduler, getScheduler);
+    JSG_LAZY_INSTANCE_PROPERTY(performance, getPerformance);
```

Comment:
If someone is currently assigning `performance` to a different value, this won't get in the way of that, right?

Similarly, if someone has a service-workers-syntax script and defines a binding called `performance`, the binding will overwrite this, right?

Reply:
It should be just fine. That's why we introduced these as lazy instance properties and do not specify that they are read-only.